tutorials/object_spark_1_intro.ipynb

```diff
@@ -0,0 +1,682 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# DC2 object Run1.1p Apache Spark tutorial -- Part I: Apache Spark access\n",
+    "\n",
+    "Author: **Julien Peloton** [@JulienPeloton](https://github.com/JulienPeloton)  \n",
+    "Based on the work of: **Francois Lanusse [@EiffL](https://github.com/LSSTDESC/DC2-analysis/issues/new?body=@EiffL), Javier Sanchez [@fjaviersanchez](https://github.com/LSSTDESC/DC2-analysis/issues/new?body=@fjaviersanchez)** using GCR.  \n",
+    "Last Verifed to Run: **2018-10-22**\n",
+    "\n",
+    "This notebook will illustrate the basics of accessing the merged object catalogs through Apache Spark as well as how to select useful samples of stars/galaxies from the dpdd catalogs (from DM outputs). It follows the same steps (and sometimes same documentation) as in this [notebook](https://github.com/LSSTDESC/DC2-analysis/blob/master/tutorials/object_gcr_1_intro.ipynb) which uses the Generic Catalog Reader ([GCR](https://github.com/yymao/generic-catalog-reader)).\n",
+    "\n",
+    "__Learning objectives__:\n",
+    "\n",
+    "After going through this notebook, you should be able to:\n",
+    "  1. Load and efficiently access a DC2 object catalog with Apache Spark\n",
+    "  2. Understand and have references for the catalog schema\n",
+    "  3. Apply cuts to the catalog using Spark SQL functionalities\n",
+    "  4. Have an example of quality cuts and simple star/galaxy separation cut\n",
+    "  5. Distribute the computation and the routine to plot to be faster!\n",
+    "\n",
+    "__Logistics__: This notebook is intended to be run through the JupyterHub NERSC [interface](https://jupyter-dev.nersc.gov) with the `desc-python-spark` kernel. If you want to setup your Spark+DESC environment at NERSC, please follow the instructions available here: [spark-kernel-nersc](https://github.com/astrolabsoftware/spark-kernel-nersc). Note that a general introduction and tutorials for Apache Spark can be found at [astrolabsoftware/spark-tutorials](https://github.com/astrolabsoftware/spark-tutorials) (under construction).\n",
```

Comment:
Update the kernel installation instruction. 
- `desc-python-spark` --> `desc-pyspark`
- User can use the setup script `/global/common/software/lsst/common/miniconda/kernels/setup.sh`